src/Microsoft.AspNetCore.SignalR.Specification.Tests/DefaultHubLifetimeManagerTests.cs

```diff
@@ -0,0 +1,20 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
```

Comment:
These should continue living in the test folder and just use the `Specification.Tests` project